style.css

```diff
@@ -149,3 +159,175 @@ html body {
 .circle-trans {
   margin-left: 30px;
 }
+
+.second-section {
+  display: grid;
+  row-gap: 5em;
+  justify-content: center;
+  margin-top: 5em;
+}
+
+.item {
+  display: grid;
+  column-gap: 5em;
+  background-color: #fff;
+  border: 0.063em solid #dfe1e6;
+  border-radius: 1em;
+  width: 20.438em;
+  justify-content: center;
+  padding-top: 0.938em;
+}
```

Comment:
- [ ] Your project cards are great and very similar to the Figma template, but the project required to use Flexbox to style each card, while using Grid to style the whole section. Kindly change the `display` to Flexbox to style each individual card to meet the project requirements.